app/Http/Transformers/PostTransformer.php

```diff
@@ -27,6 +27,7 @@ public function transform(Post $post)
                 ],
                 'caption' => $post->content->caption,
                 'status' => $post->content->status,
+                'total_reactions' => count($post->content->reactions),
```

Comment:
@DFurnes ah I think you're right! Matt helped me turn on a log in my terminal and it looks like it is still making the extra SQL query:
![screen shot 2017-03-15 at 2 15 02 pm](https://cloud.githubusercontent.com/assets/9019452/23964198/13662d6e-098a-11e7-805b-4348eb25d2e7.png)
In this case, I don't think eager loading will help since we're doing this in the Post Transformer with only 1 post at a time? 

I'm trying to think about how we can use the `::withCount` query in this case - would we want to go back to [this function](https://github.com/DoSomething/rogue/blob/master/app/Http/Transformers/SignupTransformer.php#L61-L66) and load the `$posts::withCount` here? But then we'd have to have another query here as well?